code/game/machinery/computer/law.dm

```diff
@@ -6,6 +6,7 @@
 	circuit = /obj/item/circuitboard/aiupload
 	var/mob/living/silicon/ai/current = null
 	var/opened = 0
+	var/upload_cooldown = 0
```

Comment:
This won't make sense until you see the next comments:

    var/next_upload = 0